python/tests/cassettes/test_nessie_cli_auth/test_bearer_auth.yaml

```diff
@@ -1,4 +1,155 @@
 interactions:
```

Comment:
Is re-recording "auth" cassettes necessary for this PR?

Reply:
Yes